components/slider/index.css

```diff
@@ -640,97 +641,72 @@ governing permissions and limitations under the License.
-
-      &:hover,
-      &:active {
-        border-color: var(--highcontrast-disabled-border-color, var(--mod-disabled-border-color, var(--spectrum-disabled-border-color)));
-        background: var(--highcontrast-slider-handle-background-color-disabled, var(--mod-slider-handle-background-color-disabled, var(--spectrum-slider-handle-background-color-disabled)));
-      }
+  .spectrum-Slider-handle {
+    border-color: var(--highcontrast-slider-handle-border-color-disabled, var(--mod-slider-handle-border-color-disabled, var(--spectrum-slider-handle-border-color-disabled)));
+    background: var(--highcontrast-slider-handle-disabled-background-color, var(--mod-slider-handle-disabled-background-color, var(--spectrum-slider-handle-disabled-background-color)));
+
+    &:hover,
+    &:active {
+      border-color: var(--highcontrast-disabled-border-color, var(--mod-disabled-border-color, var(--spectrum-disabled-border-color)));
+      background: var(--highcontrast-slider-handle-background-color-disabled, var(--mod-slider-handle-background-color-disabled, var(--spectrum-slider-handle-background-color-disabled)));
     }
+  }
 
-    .spectrum-Slider-track {
-      &::before {
-        background: var(--highcontrast-slider-track-color-disabled, var(--mod-slider-track-color-disabled, var(--spectrum-slider-track-color-disabled)));
-      }
+  .spectrum-Slider-track {
+    &::before {
+      background: var(--highcontrast-slider-track-color-disabled, var(--mod-slider-track-color-disabled, var(--spectrum-slider-track-color-disabled)));
     }
+  }
 
-    &.spectrum-Slider--filled {
-      .spectrum-Slider-track:first-child {
-        &::before {
-          background: var(--highcontrast-slider-track-fill-color-disabled, var(--mod-slider-track-fill-color-disabled, var(--spectrum-slider-track-fill-color-disabled)));
-        }
-      }
+  &.spectrum-Slider--filled {
+    .spectrum-Slider-track:first-child::before {
+      background: var(--highcontrast-slider-track-fill-color-disabled, var(--mod-slider-track-fill-color-disabled, var(--spectrum-slider-track-fill-color-disabled)));
     }
+  }
 
-    .spectrum-Slider-fill {
-      &::before {
-        background: var(--highcontrast-slider-track-fill-color-disabled, var(--mod-slider-track-fill-color-disabled, var(--spectrum-slider-track-fill-color-disabled)));
-      }
+  .spectrum-Slider-fill {
+    &::before {
+      background: var(--highcontrast-slider-track-fill-color-disabled, var(--mod-slider-track-fill-color-disabled, var(--spectrum-slider-track-fill-color-disabled)));
     }
+  }
 
-    .spectrum-Slider-ramp {
-      /* Slider handle turns transparent in high contrast mode for ramp */
-      @media (forced-colors: active) {
-        & + .spectrum-Slider-handle {
-            fill: ButtonFace;
-            background-color: ButtonFace;
-          }
-      }
```

Comment:
Was this section intentionally removed?

Reply:
It wasn't removed but was rather just moved down to the bottom of the file with the other forced-colors